src/components/Common/LanguageSelector.tsx

```diff
@@ -27,23 +33,19 @@ export const LanguageSelector = (props: any) => {
   );
 
   return (
-    <div className="relative flex items-center">
-      <select
-        className={cn(
-          props.className,
-          "w-full cursor-pointer appearance-none rounded-md py-2 pl-2 pr-10 focus:border-primary-500 focus:outline-hidden focus:ring-primary-500",
-        )}
-        id="language-selector"
-        name="language"
-        value={i18n.language}
-        onChange={(e: any) => handleLanguage(e.target.value)}
-      >
-        {availableLocales.map((e) => (
-          <option key={e} value={e}>
-            {LANGUAGES[e]}
-          </option>
-        ))}
-      </select>
+    <div className="relative flex items-center w-full">
+      <Select value={i18n.language} onValueChange={handleLanguage}>
+        <SelectTrigger className="focus:border-primary-500 focus:outline-none focus:ring-primary-500">
```

Comment:
we don't have any focus state for any of select component in codebase so we should be removing className from here just to be consistent with design 
```suggestion
        <SelectTrigger>
```